platform/broadcom/sonic-platform-modules-inventec/d7332/modules/pmbus_core.c

```diff
@@ -0,0 +1,2411 @@
+/*
+ * Hardware monitoring driver for PMBus devices
```

Comment:
use kernel source code.

Reply:
Done!
Use kernel source code for "i2c-mux-pca9541.c", "lpc_ich.c", "pmbus.h", "pmbus_core.c"
Please review again

Reply:
@lguohan please review again. Thanks!

Reply:
@lguohan 
We plan to support customer with TH3 platform.
Please approve the merge so we can get into the release.